experimental/generated/ecs/ecs_flat.yml

```diff
@@ -20786,6 +20829,7 @@ tls.client.supported_ciphers:
   - relation: match
     stability: experimental
   short: Array of ciphers offered by the client during the client hello.
+  synthetic_source_keep: none
```

Comment:
IIRC the order implies preference (most preferred to the least preferred) so we should keep source for this.

Reply:
thanks, I've removed this from `tls.client.supported_ciphers`